### PR TITLE
Add remote event show page [SSR-779]

### PR DESCRIPTION
**Pull request by hs-jkestenberg**
- Added a dynamic remote event show page that references all corresponding events.
- Added a new viewmodel (`RemoteEventAbstractAttributes`) to elegantly manage and process data.
- Added tests on viewmodel

### CODE REVIEW

**Review by hs-arodrigues — DISMISSED** [review]
Awesome work, good idea on the view model for the remote event attributes!

**Review by hs-psilva — DISMISSED** [review]

File: lib/que/web/viewmodels/remote_event_abstract_attributes.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    end
+
+    # Transmitter gateways have viable url (eg. 'http://accounts-contracts-api.homestars.int'),
+    # while receiver gateways are snake_case service name (eg. 'accounts_contracts_api')
+    def remote_event_gateway_base_url
+      if remote_event_is_transmitter?
+        @remote_event.gateway
+      else
+        tld = @request_base_url.split('.').last
+        "http://#{@remote_event.gateway.gsub('_', '-')}.homestars.#{tld}"
```
Comment (gerrypower) [inline lib/que/web/viewmodels/remote_event_abstract_attributes.rb]: The `http` will break when we go to `https` which is part of the future plan. Sinatra knows, and maybe you just pass it to the class from the caller with `request.scheme`?

**Review by gerrypower — CHANGES_REQUESTED** [review]
Great work! One issue I would ask to fix as noted.

**Review by gerrypower — DISMISSED** [review]

**Review by hs-arodrigues — APPROVED** [review]
Beautiful.

**Pull request merged**